cmd/maya-agent/app/exporter.go

```diff
@@ -0,0 +1,51 @@
+package app
+
+import (
+	"log"
+	"net/http"
+	"net/url"
+
+	"github.com/openebs/maya/cmd/maya-agent/app/collector"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Entrypoint is used to monitor OpenEBS volumes. It's used by maya-agent to
+// start an instance of openebs exporter.
```

Comment:
Is an instance created for every volume ? Is the instance applicable for all the volumes in the Node ? Can these be described as method level comments.

Reply:
One instance of exporter can only gathers one controller's Metrics / Stats. So you need to run n no of exporters in order to gather metrics from n volume controllers. You need to pass controller's address at run time as flag

Reply:
move the exporter.go in its own package -- maya-agent/exporter/exporter.go